README.md

```diff
@@ -130,6 +130,8 @@ By convention, the stub file for an exercise with slug `exercise-slug`
 must be named `exercise_slug.go`. This is because CI needs to delete stub files
 to avoid conflicting definitions.
 
+The track stops automatically providing stub files at the [twelve-days-exercise](exercises/twelve-days/).
```

Comment:
I'm a little uncomfortable with this wording; `The track stops automatically` gives the impression this is a hard stop that we have no control over. My understanding is that while stubs primarily exist as a helpful start in the early exercises for new Go programmers they also offer the opportunity to provide guidance in later exercises where there may be confusion, a particularly difficult concept, or boilerplate code required. If this is generally agreed then it would be good to reflect this in the README. If not then I think it's just the word `automatically` that I don't like, as this is a choice we have manually implemented, and I think it reads just as well without it, though the addition of a sentence about why stub files are no longer provided would be helpful.